addons/crm/models/crm_team_member.py

```diff
@@ -121,7 +121,8 @@ def _assign_and_convert_leads(self, work_days=1):
             )
 
         members_data, population, weights = dict(), list(), list()
-        members = self.filtered(lambda member: not member.assignment_optout and member.assignment_max > 0)
+        members = self.filtered(lambda member: not member.assignment_optout and member.assignment_max > 0).sorted('id')
```

Comment:
It will more clean to change the default order of the model (`crm.team.member`) instead, no?

Reply:
Yes you are right! ;)